tests/rkt_image_rm_test.go

```diff
@@ -265,12 +357,15 @@ func getImageId(ctx *testutils.RktRunCtx, name string) (string, error) {
 	return imageID, nil
 }
 
-func removeImage(ctx *testutils.RktRunCtx, image string, shouldWork bool) error {
-	expect := fmt.Sprintf(rmImageReferenced, image)
+func removeImage(ctx *testutils.RktRunCtx, images []string, shouldWork bool) error {
+	cmd := fmt.Sprintf("%s image rm", ctx.Cmd())
```

Comment:
You could use `strings.Join(images, " ")` instead of the for loop below - `fmt.Sprintf("%s image rm %s", ctx.Cmd(), strings.Join(...))`.